.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: "Build"
+on: push
```

Comment:
limit to `branches: [master]`

Reply:
I think I'm not on the same page as you here. The current behavior is to build in a precommit step so that all branches have an up-to-date dist/index.js. Why should we only build some branches?